app/Resources/views/base.html.twig

```diff
@@ -2,26 +2,21 @@
 <html>
     <head>
         <meta name="viewport" content=" minimum-scale=1.0, maximum-scale=1.0, width=device-width, user-scalable=no">
-        <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+        <meta charset="utf-8" />
+
         {{ sonata_seo_metadatas() }}
         {{ sonata_seo_link_canonical() }}
         {{ sonata_seo_lang_alternates() }}
         {{ sonata_seo_title() }}
 
         {% block includes %}
-            {% stylesheets
-                '@AppBundle/Resources/public/bootstrap/css/bootstrap.min.css'
-                '@AppBundle/Resources/public/bootstrap/css/bootstrap-responsive.min.css'
-                '@AppBundle/Resources/public/css/style.css'
-            %}
-
-            <!-- Le HTML5 shim, for IE6-8 support of HTML5 elements -->
+            <!-- HTML5 shim, for IE6-8 support of HTML5 elements -->
             <!--[if lt IE 9]>
-            <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
+            <script src="{{ asset("vendor/html5shiv/dist/html5shiv.js") }}"></script>
```

Comment:
IE8? we might just ditch that then, i think. or is it relevant for less antique browsers as well?